packages/orbit-components/src/NavigationBar/index.tsx

```diff
@@ -64,12 +70,17 @@ const NavigationBar = ({
       data-test={dataTest}
       id={id}
       className={cx(
-        "bg-white-normal p-300 z-navigation-bar fixed inset-x-0 top-0 box-border flex w-full translate-x-0 items-center",
-        "duration-normal transform-gpu transition-transform ease-in-out",
+        "p-300 z-navigation-bar fixed inset-x-0 top-0 box-border flex w-full translate-x-0 items-center",
+        "duration-normal transform-gpu ease-in-out",
+        transparentBgAtTop
+          ? "transition-[transform_background-color_border-color]"
```

Comment:
Isn't this declaration missing commas? 🤔 

![Snímek obrazovky 2024-12-09 v 12 48 30](https://github.com/user-attachments/assets/ede8004b-90cd-4235-9a52-0c619a65059d)

Reply:
Good catch, thank you! Fixing